Makefile

```diff
@@ -146,11 +144,6 @@ else ifeq ($(TARGET_NAME),TARGET_STAX)
 SDK_SOURCE_PATH  += lib_blewbxx lib_blewbxx_impl
 endif
 
-# If the SDK supports Flow for Nano S, build for it
-ifeq ($(TARGET_NAME),TARGET_NANOS)
-DEFINES += HAVE_WALLET_ID_SDK
-endif
-
 # U2F
 DEFINES   += U2F_PROXY_MAGIC=\"TRX\"
 DEFINES   += HAVE_IO_U2F HAVE_U2F
```

Comment:
Not sure to understand why you mentioned "U2F" in the PR title. But there are still some U2F stuff in your makefile (after  your changes)